src/components/ec-panel/ec-panel.vue

```diff
@@ -5,14 +5,27 @@
   >
     <div class="ec-panel__header">
       <div class="ec-panel__header-icons">
+        <a
+          v-if="isBackEnabled"
+          aria-label="Go back"
+          class="ec-panel__header-action ec-panel__header-action--back"
+          href="#"
+          @click.stop.prevent="goBack"
+        >
+          <ec-icon
+            class="ec-panel__header-icon"
+            name="simple-chevron-left"
+            :size="24"
+          />
+        </a>
         <a
           aria-label="Close panel"
-          class="ec-panel__close"
+          class="ec-panel__header-action ec-panel__header-action--close"
```

Comment:
OT: While you are here, could you please add data-test on everything clickable in this component? We have lots of references to these classes in integration tests and unit tests, we should replace them with data-test.

Reply:
Well spotted. Done.